### PR TITLE
chore: add datePicker in storybook (CALCOM-10760)

### PR DESCRIPTION
**Pull request by gitstart-calcom**
## Demo
https://www.loom.com/share/d610522754bd4e39bca40576a090d0b9

### CODE REVIEW

**Review by sean-brydon — CHANGES_REQUESTED** [review]
Judging with how this component is styled. I believe this is the wrong date picker. 

I did a quick look at refernces and this appears to be dead component that we dont use anywhere

**Comment by gitstart-calcom** [comment]
Hi @sean-brydon please can you help us to find the correct file?

**Comment by PeerRich** [comment]
definitely wrong component. should we close this PR and open a new one or do you wanna edit this?

**Comment by sean-brydon** [comment]
> Hi @sean-brydon please can you help us to find the correct file?

Hi the correct file we use for date picker is located at

cal.com/packages/features/calendars/DatePicker.tsx

**Comment by gitstart-calcom** [comment]
> definitely wrong component. should we close this PR and open a new one or do you wanna edit this?

We have already updated to the right component in the same PR. We will be pushing the changes shortly

**Comment by gitstart-calcom** [comment]
Hi @sean-brydon we made the requested changes here, please take a look

File: packages/features/calendars/DatePicker.tsx [hunk 1]
```diff
@@ -69,7 +69,11 @@ export const Day = ({
       {...props}>
       {date.date()}
       {date.isToday() && (
-        <span className="absolute left-1/2 top-1/2 flex h-[5px] w-[5px] -translate-x-1/2 translate-y-[8px] items-center justify-center rounded-full bg-white align-middle sm:translate-y-[12px]">
+        <span
+          className={classNames(
+            "bg-brand-default absolute left-1/2 top-1/2 flex h-[5px] w-[5px] -translate-x-1/2 translate-y-[8px] items-center justify-center rounded-full align-middle sm:translate-y-[12px]",
+            active && "invert"
```
Comment (gitstart-calcom) [inline packages/features/calendars/DatePicker.tsx:75]: We noticed some style issues and fixed them too. Please checkout the video for more detail
https://www.loom.com/share/ed35d62be2d34f199c2af8796b467cce

**Review by sean-brydon — APPROVED** [review]
Great job :) Thanks for the prompt fixes. Nice spot on the active state too

**Pull request merged**